packages/esm-appointments-app/src/config-schema.ts

```diff
@@ -31,6 +31,11 @@ export const configSchema = {
     _description: 'Open links in Bahmni Appointments UI instead of O3 UI',
     _default: false,
   },
+  excludeStatuses: {
```

Comment:
In general I think it is better to have a positive mask than a negative one. So that would be changing this from `excludeStatuses` to `shownStatuses` and then have the default be `['Check In', 'Complete']`.

Also, this seems a little weird to be set via configuration file. I can imagine a user wanting to toggle through all statuses themselves. I.e. looking for a patient who has canceled or missed and then going back to normal 'check in' view. If you are filtering in the configuration then the users won't be able to toggle the filters later themselves